docker/Dockerfile

```diff
@@ -17,3 +19,15 @@ RUN install2.r --error remotes &&  \
 
 COPY . /src
 RUN R CMD INSTALL --install-tests /src && rm -rf /src
+
+ENV ORDERLY_RUNNER_QUEUE_ID=orderly.runner
+ENV REDIS_CONTAINER_NAME=orderly.runner_redis
+
+COPY docker/bin /usr/local/bin/
+
+RUN git config --global --add safe.directory /orderly-root
+RUN echo ".packit" > /.gitignore
+RUN git config --global core.excludesFile "/.gitignore"
```

Comment:
same issue as in outpack_server PR, gotta add this as a safe directory as it is not owned by us in the container, also the global gitignore means that the .packit where the workers are isnt tracked by git